action.yml

```
@@ -35,12 +35,20 @@ inputs:
       Default is added.
     default: 'added'
     required: false
+  fail_level:
+    description: |
+      The level of failures that will cause the action to fail [any,info,warning,error]
+      Default is error.
+    default: ''
+    required: false
   fail_on_error:
     description: |
+      **Deprecated**. This option is no longer recommended for use and will be removed in future versions.
       Exit code for reviewdog when errors are found [true,false]
       Default is `false`.
     default: 'false'
     required: false
+    deprecated: true
```

`deprecated` is invalid.
Please use `deprecationMessage` instead of it.

```suggestion
    deprecationMessage: Deprecated, use `fail_level` instead.
```

see also.

* https://github.com/reviewdog/action-rubocop/blob/master/action.yml#L18
* https://docs.github.com/ja/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage

---

Thanks!